app/assets/javascripts/edit_metadata.js

```diff
@@ -20,6 +21,7 @@ chf.autocompletes = chf.autocompletes || {
   generic_work_place_of_publication: '/authorities/search/assign_fast/all',
   generic_work_place_of_creation: '/authorities/search/assign_fast/all',
   batch_edit_item_subject: '/authorities/search/assign_fast/all',
+  batch_edit_item_after:          '/authorities/search/assign_fast/all',
   batch_edit_item_artist:          '/authorities/search/assign_fast/all',
   batch_edit_item_author:          '/authorities/search/assign_fast/all',
   generic_work_addressee:          '/authorities/search/assign_fast/all',
```

Comment:
Huh, i would not have known about this file, good to know! 

By autocompleting from 'assign_fast/all', does that mean someone _could_ autocomplete a subject as a creator/maker?